spec/lib/deploy/activate_spec.rb

```diff
@@ -62,8 +64,8 @@
 
       # top-level key from application.yml.example
       expect(combined_application_yml['trusted_ca_root_identifiers']).not_to be_empty
-      # four fingerprints, each of length 59, separated by three commas
-      expect(combined_application_yml['trusted_ca_root_identifiers'].length).to eq(239)
+      # six fingerprints, each of length 59, separated by four commas
+      expect(combined_application_yml['trusted_ca_root_identifiers'].length).to eq(TRUSTED_ROOT_COUNT * 59 + TRUSTED_ROOT_COUNT - 1)
```

Comment:
TRUSTED_ROOT_COUNT * 60 - 1

Reply:
Yeah. I was trying to keep the correspondence with the comment, even if I wasn't going with the same numbers. I'll change it to `___` in the comments next time I add a trusted root.